<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Management</class>
 <widget class="QWidget" name="Management">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_photo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_username">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>80</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Management/src/USERNAME_W_55_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_gametime">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>128</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Management/src/GAMETIME_W_60_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_horsehave">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>176</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Management/src/HORSEHAVE_W_60_25.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changename">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Management/src/CHANGEUSERNAME_W.png)}
QPushButton:hover{border-image: url(:/Management/src/CHANGEUSERNAME_P.png)}
QPushButton:pressed{border-image:url(:/Management/src/CHANGEUSERNAME_C.png) }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changecode">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Management/src/CHANGECODE_W.png)}
QPushButton:hover{border-image:url(:/Management/src/CHANGECODE_P.png) }
QPushButton:pressed{border-image:url(:/Management/src/CHANGECODE_C.png) }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Return/src/RETURN_W.png)}
QPushButton:hover{border-image:url(:/Return/src/RETURN_P.png)}
QPushButton:pressed{border-image: url(:/Return/src/RETURN_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_toprank">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Management/src/TOPRANK_W_120_50.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_topscore">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Management/src/TOPSCORE_W_120_50.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_toprank">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>250</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_topscore">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_horsehave">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>75</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_gametime">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>123</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_up">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Management/src/LASTINDEX_UP_W.png)}
QPushButton:hover{border-image:url(:/Management/src/LASTINDEX_UP_P.png) }
QPushButton:pressed{border-image:url(:/Management/src/LASTINDEX_UP_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_down">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Management/src/NEXTINDEX_DOWN_W.png)}
QPushButton:hover{border-image:url(:/Management/src/NEXTINDEX_DOWN_P.png) }
QPushButton:pressed{border-image:url(:/Management/src/NEXTINDEX_DOWN_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_switchEH">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Management/src/LASTINDEX_LEFT_W.png)}
QPushButton:hover{border-image:url(:/Management/src/LASTINDEX_LEFT_P.png) }
QPushButton:pressed{border-image:url(:/Management/src/LASTINDEX_LEFT_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Management/src/NEXTINDEX_RIGHT_W.png)}
QPushButton:hover{border-image:url(:/Management/src/NEXTINDEX_RIGHT_P.png) }
QPushButton:pressed{border-image:url(:/Management/src/NEXTINDEX_RIGHT_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="index.qrc"/>
 </resources>
 <connections/>
</ui>
